<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForceFactor</class>
 <widget class="QDialog" name="ForceFactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lePower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leObxvat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFullPower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="leMono">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>угол обхвата</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="leDuo1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDuo2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>240</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMono">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>однобарабанный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbDuo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>двубарабанный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMulti">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>многобарабанный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbHands">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ручной</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ForceFactor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ForceFactor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
